index.yaml

```diff
@@ -796,6 +796,30 @@ entries:
     - assets/rancher-alerting-drivers/rancher-alerting-drivers-1.0.100.tgz
     version: 1.0.100
   rancher-backup:
+  - annotations:
+      catalog.cattle.io/auto-install: rancher-backup-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Rancher Backups
+      catalog.cattle.io/namespace: cattle-resources-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/provides-gvr: resources.cattle.io.resourceset/v1
+      catalog.cattle.io/release-name: rancher-backup
+      catalog.cattle.io/scope: management
+      catalog.cattle.io/ui-component: rancher-backup
```

Comment:
Needs a max of rancher 2.5.99 since we have split branch lines. I don't think we want a min here because we purposefully want this going back to earlier 2.5.x versions.

Reply:
the annotation will be added in the rancher-backup 1.2.0-rc5
https://github.com/rancher/backup-restore-operator/pull/155

Reply:
The annotation is in the chart now!